src/remove-dulplicates/index.js

```diff
@@ -1,3 +1,18 @@
-function removeDuplicates(obj) {}
+function removeDuplicates(obj) {
+    let keys = Object.keys(obj)
+    let stack = []
+    for (let index = keys.length - 1; index >= 0; index--) {
+        let key = keys[index] 
+        let temp = obj[key] 
+        obj[key] = temp.filter((el) => {
+            if (stack.indexOf(el) < 0) {
```

Comment:
`stack.indexOf(el) !== -1` preferred, or even better, since you don't need the index, `!stack.includes(el)` and you can even flip the condition that way.